src/node/internal/internal_utils.ts

```diff
@@ -49,11 +50,13 @@ export function slowCases(enc: string): Encoding | undefined {
     case 4:
       if (enc === 'UTF8') return UTF8;
       if (enc === 'ucs2' || enc === 'UCS2') return UTF16LE;
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-template-expression
       enc = `${enc}`.toLowerCase();
```

Comment:
Why is this needed? enc is supposed to be a string.

Also `ucs2` is repeated l52 and l56